src/index.ts

```diff
@@ -465,6 +465,22 @@ function liskAddressDecoder(data: string): Buffer {
 
   return Buffer.from(bigInt(data.slice(0, -1)).toString(16), 'hex');
 }
+  
+function seroAddressEncoder(data: Buffer): string {
+  const address =  bs58EncodeNoCheck(data);
+
+  return address;
+}
+
+function seroAddressDecoder(data: string): Buffer {
+  if (/^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]+$/i.test(data)) {
```

Comment:
Are you doing if the string is bs58 or not? Does `bs58Encode/Decode` do that already?

Reply:
Are you doing if the string is bs58 or not? Yes, I am using bs58. Thanks I have cleaned up my code.